output/mapping/F02_2014.csv

```diff
@@ -27,7 +27,7 @@ xpath,label-key,index,comment,guidance
 /CONTRACTING_BODY/URL_DOCUMENT,,,,Map to the party's `.contactPoint.url`
 /CONTRACTING_BODY/ADDRESS_FURTHER_INFO_IDEM,as_contacts_above,,,Discard. Additional information can be obtained from the abovementioned address unless otherwised specified.
 /CONTRACTING_BODY/ADDRESS_FURTHER_INFO,address_another,,,"If only the contact person is different, use the [Additional Contact Points](https://github.com/open-contracting-extensions/ocds_additionalContactPoints_extension) extension, get the `Organization` object for the buyer, add a `ContactPoint` object to its `.additionalContactPoints` array, and set its [fields as above](#/CONTRACTING_BODY/ADDRESS_CONTRACTING_BODY/CONTACT_POINT).\n\nOtherwise, [add a party](../operations#add-a-party), add 'processContactPoint' to its `.roles`, and set its [fields as above](#/CONTRACTING_BODY/ADDRESS_CONTRACTING_BODY/OFFICIALNAME). (PROPOSAL #9)"
-/CONTRACTING_BODY/URL_PARTICIPATION,address_send_tenders,,,"Add 'electronicSubmission' to `tender.submissionMethod`, and map to `tender.submissionMethodDetails`\n\nIf *to the following address* is selected, this results in a loss of structure. (WARNING #27)"
+/CONTRACTING_BODY/URL_PARTICIPATION,address_send_tenders,,,"Add 'electronicSubmission' to `tender.submissionMethod` and map to `tender.submissionMethodDetails` `\n\nSet `tender.submissionTerms.electronicSubmissionPolicy` to ‘required’\n\nIf *to the following address* is selected, this results in a loss of structure. (WARNING #27)"
```

Comment:
Looks like there's an extra space and backtick at the beginning of the new content.

Can you use regular single quotes instead of curly quotes, in all cases?

Reply:
That must be LibreOffice that applied some regional setting, because I have to way of doing curly quote with my keyboard. I'll fix it and make sure it doesn't happen anymore.

Reply:
And, I noticed you deleted a comma. I prefer to keep commas because clauses, because it makes it easier for the reader to separate the steps.

Reply:
What about more line breaks?

Reply:
I agree that if the text is on the same line, commas help.

Reply:
Is there a reason why we don't separate each instruction with line breaks? That's perfect for clarity. Even better: we could number the instructions! I'm OK to do it, I love numbered instructions :)

Reply:
It's a good idea, but I think that's a lot of work to redo all the mappings. Let's put it as an issue that maybe we'll address once we're done all the mappings. If we consistently use commas and re-use sentence constructions, it should be semi-automatable.

Reply:
Issue opened: https://github.com/open-contracting/european-union-support/issues/11